Tests/UnitTests/Mocks/MockSimpleCache.swift

```diff
@@ -17,6 +17,8 @@ import Foundation
 class MockSimpleCache: LargeItemCacheType, @unchecked Sendable {
 
     var cacheDirectory: URL?
+    var workingCacheDirectory: URL?
+    var workingDocsDirectory: URL?
```

Comment:
Why are these two needed?

Reply:
The cache sets the working directory once when used by the consumer. This is used to test that the clear happens at the working directory.